src/control/lib/control/pool.go

```diff
@@ -909,18 +892,23 @@ func PoolDrain(ctx context.Context, rpcClient UnaryInvoker, req *PoolRanksReq) (
 		return nil, err
 	}
 	if msErr := ur.getMSError(); err != nil {
```

Comment:
Found one actual bug. I think this issue needs to be fixed before landing.

```suggestion
	if msErr := ur.getMSError(); msErr != nil {
```

Reply:
done